src/utils/auth/api.ts

```diff
@@ -29,7 +29,7 @@ export const privateFetcher = async <T>(input: RequestInfo, init?: RequestInit):
  * @param {RequestData} requestData
  * @returns {Promise<T>}
  */
-const postRequest = <T>(url: string, requestData: RequestData): Promise<T> =>
+export const postRequest = <T>(url: string, requestData: RequestData): Promise<T> =>
```

Comment:
This shouldn't be exported and instead we can export the function that does the actual calling.